encoding/toml.ts

```diff
@@ -40,15 +40,18 @@ class Parser {
   }
 
   _mergeMultilines(): void {
+    /** @private */
     function arrayStart(line: string): boolean {
       const reg = /.*=\s*\[/g;
       return reg.test(line) && !(line[line.length - 1] === "]");
     }
 
+    /** @private */
```

Comment:
All of these @private comments are superfluous. Either the method is public and exported or it's private. How can we adding these superfluous comments?

Reply:
Agree with you and i've not found any workaround (for now) for this problem. There is only this in the doc: https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-settings-allow-private-to-disable-rules-for-that-comment-block

Problem is this is the reference plugin from eslint : https://eslint.org/blog/2018/11/jsdoc-end-of-life and it seems a bit incomplete for the private part.

Reply:
Here’s the issue: https://github.com/gajus/eslint-plugin-jsdoc/issues/192

Reply:
GREAT